### DISCUSSION TITLE
Upgrade from RS

### DISCUSSION THREAD

**Discussion by jfreak53** [dani-garcia/vaultwarden#3086] [Q&A]
What would be the best way to upgrade to vaultwarden from bitwarden_rs server? Been searching, might already exist, but I'm not finding.

**Comment by BlackDex** [comment]
Just change `bitwarden_rs/server` to `vaultwarden/server` 😉 

Also, check the big banner at the top of the discussions page: https://github.com/dani-garcia/vaultwarden/discussions/1642
